aspnetcore/fundamentals/middleware/index/snapshot/Program80ShortCircuit.cs

```diff
@@ -0,0 +1,12 @@
+var app = WebApplication.Create();
+
+app.UseHttpLogging();
+
+// <mapget>
+app.MapGet("/short-circuit", () => "Hello world!").ShortCircuit();
```

Comment:
How about adding another endpoint, /no-short-circuit so they can easily compare the logging?

Reply:
Add `/` so they don't get an error when they launch. Don't add short circuit to it.